Sources/LiveViewNative/Coordinators/LiveViewCoordinator.swift

```diff
@@ -298,6 +300,28 @@ public class LiveViewCoordinator<R: RootRegistry>: ObservableObject {
         }
     }
     
+    private func getPlatformMetadata() throws -> String {
+        let encoder = JSONEncoder()
+        let metadata: [String : String] = [
+            "platform_variant": getPlatformVariant()
+        ]
+        let encodedMetadata = try encoder.encode(metadata)
+
+        return String(data: encodedMetadata, encoding: .utf8)!
```

Comment:
Is there a reason this needs to be JSON-encoded inside the connectParams? The return type of getPlatformMetadata could just be `Payload` (i.e., `[String: Any]`) and the dictionary could be returned directly

Reply:
Thanks for the suggestion. I made that change and it simplifies the use of platform metadata from the Elixir side as well. 🙂